index.html

```diff
@@ -6,6 +6,11 @@ <h2 id="exasol-document-mapping-language-edml">Exasol Document Mapping Language
 <li><a href="edml-1.1.0.json">1.1.0 (current)</a> (<a href="https://exasol.github.io/virtual-schema-common-document/schema_doc/edml_1.1.0/index.html">reference</a>)</li>
 <li><a href="edml-1.0.0.json">1.0.0</a> (<a href="https://exasol.github.io/virtual-schema-common-document/schema_doc/edml_1.0.0/index.html">reference</a>)</li>
 </ul>
+<h2 id="exasol-error-code-report">Exasol Error Code Report</h2>
+<p>The Exasol error code report is a JSON report that is generated by local crawlers from the source code. It list all error codes defined in the project together with some metadata. The report format is standardized by this JSON-Schema.</p>
+<ul>
+<li><a href="error_code_report-0.1.0.json">0.1.0 (current)</a> <a href="https://exasol.github.io/schemas/error_code_report-0.1.0/index.html">(reference)</a> <a href="error_code_report-0.1.0_example.json">(example)</a></li>
```

Comment:
it might make sense to include the example link here too

Reply:
yes, but relatice links don't work here and since it's not in master I can't use an absolute one...